pkg/apis/featuretoggle/v0alpha1/types.go

```diff
@@ -46,6 +46,9 @@ type FeatureSpec struct {
 
 	// Do not show the value in docs
 	HideFromDocs bool `json:"hideFromDocs,omitempty"`
+
+	// Expression to determine if the flag is enabled by default
+	Expression string `json:"expression,omitempty"`
```

Comment:
i might be missing something obvious, but why is this field called `expression`?

Reply:
historic -- and this will all go away with the work @hairyhenderson is doing.

Originally this supported CEL expressions, but then we were never quite sure where/how to set them